stdlib/asyncio/taskgroups.pyi

```diff
@@ -0,0 +1,17 @@
+import sys
+from types import TracebackType
+from typing import Any, Coroutine, Generator, TypeVar
+
+from .tasks import Task
+
+_T = TypeVar("_T")
+
+if sys.version_info >= (3, 11):
+    class TaskGroup:
+        def __init__(self, name: str | None = ...) -> None: ...
+        def get_name(self) -> str: ...
+        async def __aenter__(self) -> TaskGroup: ...
```

Comment:
When called on a subclass of `TaskGroup`, it will return an instance of the subclass. While PEP 673 isn't available for us yet, we use a typevar defined in `_typeshed` instead:

```suggestion
        async def __aenter__(self: Self) -> Self: ...
```

You need to also add `from _typeshed import Self`.

Reply:
Whoops, didn't see this before. Fixed now!